elyra/cli/pipeline_app.py

```diff
@@ -151,6 +152,13 @@ def _preprocess_pipeline(pipeline_path: str, runtime: str, runtime_config: str)
 
 def _execute_pipeline(pipeline_definition):
     try:
+        # validate pipeline
+        validation_response = asyncio.get_event_loop().run_until_complete(
+            PipelineValidationManager.instance().validate(pipeline=pipeline_definition))
+        if validation_response.has_fatal:
+            for issue in validation_response.to_json().get('issues'):
+                print(f'>>> {issue}')
```

Comment:
Should this be replaced with `print_info()` or something similar?  At a minimum, the set of issues should have a header indicating a validation failure occurred.

Reply:
Agreed. Should stop processing when a fatal flag is detected.